app/res/layout/screen_login.xml

```diff
@@ -22,9 +22,11 @@
     </LinearLayout>
 
     <LinearLayout
+        android:id="@+id/main_pane"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
+        android:paddingTop="55sp"

```

Comment:
hm, this doesn't seem right. This is spacing down the view? 

Can we not float this somewhere without pushing it down manually? What happens when this screen is very large and very small?